eng/targets/Helix.Common.props

```diff
@@ -11,20 +12,30 @@
     <HelixAvailablePlatform Include="OSX" />
     <HelixAvailablePlatform Include="Linux" />
 
-    <HelixAvailableTargetQueue Include="Windows.10.Amd64.ClientRS4.VS2017.Open" Platform="Windows" />
-    <HelixAvailableTargetQueue Include="Windows.81.Amd64.Open" Platform="Windows" />
-    <HelixAvailableTargetQueue Include="Windows.7.Amd64.Open" Platform="Windows" />
-    <HelixAvailableTargetQueue Include="Windows.10.Amd64.EnterpriseRS3.ASPNET.Open" Platform="Windows" EnableByDefault="false" />
-    <HelixAvailableTargetQueue Include="OSX.1013.Amd64.Open" Platform="OSX" />
-    <HelixAvailableTargetQueue Include="Ubuntu.1604.Amd64.Open" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="Ubuntu.1804.Amd64.Open" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="Centos.7.Amd64.Open" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="Debian.8.Amd64.Open" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="Debian.9.Amd64.Open" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="Redhat.7.Amd64.Open" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="(Fedora.28.Amd64.Open)Ubuntu.1604.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-28-helix-09ca40b-20190508143249" Platform="Linux" />
-
-    <!--  TODO: re-enable Debian.9.Arm64.Open and Ubuntu.1804.Arm64.Open    -->
+    <HelixAvailableArch Include="x64" />
+    <HelixAvailableArch Include="arm64" />
+    
+    <!-- x64 queues -->
+    <HelixAvailableTargetQueue Include="Windows.10.Amd64.ClientRS4.VS2017.Open" Platform="Windows" Condition="'$(TargetArchitecture)' == 'x64'"/>
```

Comment:
@aspnet/build do you guys have any suggestions for better msbuild magic to accomplish this filtering for arm vs x64?

Reply:
Seems reasonable as-is. What would you like to improve?

Reply:
Seems different than the other attribute pattern used for platform and enablebydefault, I guess maybe I can just move each arch into its own item group with a condition which will make it feel less clunky.  Just wasn't sure if there was a better pattern to do something like this